waimak-storage/src/test/scala/com/coxautodata/waimak/storage/TestAuditTableFile.scala

```diff
@@ -204,28 +208,47 @@ class TestAuditTableFile extends SparkAndTmpDirSpec {
       val onlyColdRegion_empty = AuditTableFile.inferRegionsWithStats(sparkSession, table.storageOps, basePath, Seq(tableName), false).sortBy(_.store_region)
       onlyColdRegion_empty should be(Seq.empty)
 
-      val compactedTable = table_s2.compact(lastTS_3).get
+      val compactedTable = table_s2.compact(lastTS_3, d3d).get
 
       val onlyColdRegion = AuditTableFile.inferRegionsWithStats(sparkSession, table.storageOps, basePath, Seq(tableName), false).sortBy(_.store_region)
-      onlyColdRegion should be(Seq(AuditTableRegionInfo("person", "cold", "4", lastTS_3, false, 8, lastTS_2)))
+      onlyColdRegion should be(Seq(AuditTableRegionInfo("person", "cold", "3", lastTS_3, false, 8, lastTS_2)))
 
       val (table_s31, cs3) = compactedTable.append(r3Data, lastUpdated(r3Data), lastTS_3).get
       val table_s3 = table_s31.asInstanceOf[AuditTableFile]
       cs3 should be(3)
 
       table_s3.regions.size should be(2)
-      table_s3.regions(0).store_region should be("4") // first compaction would re-compact first cold again
-      table_s3.regions(1).store_region should be("5")
+      table_s3.regions(0).store_region should be("3") // first compaction would re-compact first cold again
```

Comment:
The comment "first compaction would re-compact first cold again" doesn't apply now, right?

Reply:
Done